src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/logs/logs.component.html

```diff
@@ -0,0 +1,23 @@
+<div *ngIf="contentData"
+     class="container-fluid">
+  <cd-info-panel i18n *ngIf="contentData.clog">
```

Comment:
`cd-info-panel` should be used only to inform the user that there are no logs.
If there are logs, they should be listed inside the `well`

Reply:
Is it ok to use the tabset similar to the previous user interface?

Reply:
I think so.